browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -1840,7 +1840,7 @@ L.Control.JSDialogBuilder = L.Control.extend({
 			L.DomUtil.addClass(fixedtext, 'ui-text');
 		}
 
-		if (data.hidden)
+		if (data.hidden || data.visible === 'false' || data.visible === false)
```

Comment:
this should already happen in `postProcess`, is that label inside buttonbox maybe?

Reply:
I added fix: https://github.com/CollaboraOnline/online/pull/5984